packages/block-editor/src/store/selectors.js

```diff
@@ -1210,6 +1210,23 @@ export function hasSelectedInnerBlock( state, clientId, deep = false ) {
 	);
 }
 
+/**
+ * Returns true if one of the block's inner blocks is dragged.
+ *
+ * @param {Object}  state    Editor state.
+ * @param {string}  clientId Block client ID.
+ * @param {boolean} deep     Perform a deep check.
+ *
+ * @return {boolean} Whether the block has an inner block dragged
+ */
+export function hasDraggedInnerBlock( state, clientId, deep = false ) {
```

Comment:
We can also do this inline in the `useSelect` callback and avoid introducing a new selector for just one use case.

Here's a similar example:

https://github.com/WordPress/gutenberg/blob/d19dbd689b9c0885f57a2ddc422318cf4a75c276/packages/edit-site/src/components/save-button/index.js#L23-L25

Reply:
I want this new selector, it's as useful as `hasSelectedInnerBlock` and this is a perfect opportunity to introduce it via using it.

Reply:
Just sharing my personal preference of avoiding adding selector to packages scope, when I can :)